lib/matplotlib/tests/test_table.py

```diff
@@ -182,3 +182,14 @@ def test_auto_column():
     tb4.auto_set_font_size(False)
     tb4.set_fontsize(12)
     tb4.auto_set_column_width("-101")
+
+
+def test_table_cells():
+    fig = plt.figure()
+    ax = fig.add_subplot(1, 1, 1)
+    table = Table(ax)
+
+    cell = table.add_cell(1, 2, 1, 1)
+    assert isinstance(cell, CustomCell)
+    table[2, 1] = cell
```

Comment:
This should probably also be a new `CustomCell` instance to be sure that the table actually changed.